packages/plugins/i18n/package.json

```diff
@@ -24,12 +24,27 @@
     "test:unit": "jest --verbose"
   },
   "dependencies": {
+    "@strapi/design-system": "1.6.3",
+    "@strapi/icons": "1.6.3",
     "@strapi/utils": "4.6.1",
-    "lodash": "4.17.21"
+    "axios": "1.2.2",
+    "formik": "2.2.9",
+    "immer": "9.0.19",
+    "lodash": "4.17.21",
+    "prop-types": "^15.7.2",
+    "qs": "6.11.0",
+    "react": "^17.0.2",
+    "react-intl": "6.2.7",
+    "react-query": "3.24.3",
+    "react-redux": "8.0.5",
+    "react-router-dom": "5.3.4",
+    "styled-components": "5.3.3",
+    "yup": "^0.32.9"
   },
   "devDependencies": {
     "@testing-library/react": "12.1.4",
-    "msw": "1.0.0"
+    "msw": "1.0.0",
+    "redux": "^4.2.1"
```

Comment:
react-redux requires a peer dep of redux, so i would actually add this to the `dependencies` :)

Reply:
Done.